TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile

```diff
@@ -53,12 +53,10 @@ RUN pipenv install --deploy
 WORKDIR /root
 
 # Console setup bash completion and venv for interactive access
-RUN echo '. /.venv/bin/activate' >> /etc/profile.d/venv.sh
 RUN dnf install -y bash-completion
-RUN echo '. /etc/profile.d/bash_completion.sh' >> ~/.bashrc && \
-    echo '. /etc/profile.d/venv.sh' >> ~/.bashrc
-# Set ENV to control startup script in /bin/sh mode
-ENV ENV=/root/.bashrc
+RUN echo '. /etc/profile.d/bash_completion.sh' >> ~/.bashrc
+
+ENV PATH="/.venv/bin:$PATH"
```

Comment:
Discussed offline. While this will enable all container execution to resolve to this python, it prevents interactive sessions from understanding what's going on and perform actions like `deactivate`.

I've put together a PR that modifies all `docker exec` and `ecs execute-command` instances https://github.com/opensearch-project/opensearch-migrations/pull/844